src/test/java/nextstep/subway/auth/acceptance/AuthAcceptanceTest.java

```diff
@@ -1,24 +1,113 @@
 package nextstep.subway.auth.acceptance;
 
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
+import nextstep.subway.auth.dto.TokenRequest;
+import nextstep.subway.member.domain.Member;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 
+import static nextstep.subway.member.MemberAcceptanceTest.회원_등록되어_있음;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("로그인 시도")
 public class AuthAcceptanceTest extends AcceptanceTest {
+  private Member 로그인_시도_회원;
+  private Member 로그인_시도_실패_회원;
+
+  @Override
+  @BeforeEach
+  public void setUp() {
+    super.setUp();
+    로그인_시도_회원 = new Member("devsigner9920@gmail.com", "password", 27);
```

Comment:
```suggestion
    로그인_시도_성공_회원 = new Member("devsigner9920@gmail.com", "password", 27);
```

으로 작성해주시는게 보기에 더 명확할 것 같습니다 :)

Reply:
아하! 넵 명확한 픽스쳐를 작성하도록 하겠습니다 ㅎ